packages/babel-plugin-transform-react-jsx/test/fixtures/react/should-avoid-wrapping-in-extra-parens-if-not-needed/output.js

```diff
@@ -1,4 +1,16 @@
-var x = React.createElement("div", null, React.createElement(Component, null));
-var x = React.createElement("div", null, props.children);
-var x = React.createElement(Composite, null, props.children);
-var x = React.createElement(Composite, null, React.createElement(Composite2, null));
+var x =
+/*#__PURE__*/
+React.createElement("div", null,
+/*#__PURE__*/
+React.createElement(Component, null));
+var x =
+/*#__PURE__*/
+React.createElement("div", null, props.children);
+var x =
+/*#__PURE__*/
+React.createElement(Composite, null, props.children);
+var x =
+/*#__PURE__*/
+React.createElement(Composite, null,
+/*#__PURE__*/
+React.createElement(Composite2, null));
```

Comment:
The output code would be a little bit more readable and less verbose if the comment would not be in a separate line but in the same line:

``` js
var x =
/*#__PURE__*/
React.createElement(Composite, null,
/*#__PURE__*/
React.createElement(Composite2, null));
```

vs

``` js
var x = /*#__PURE__*/ React.createElement(Composite, null, /*#__PURE__*/ React.createElement(Composite2, null));
```

But this isn't important...

Reply:
I agree, I'll do it in a separate PR since it isn't strictly related to adding the annotation to JSX elements.